package.json

```diff
@@ -17,7 +17,10 @@
   "scripts": {
     "start": "grunt",
     "test": "grunt test",
-    "postinstall": "bower install --config.interactive=false"
+    "postinstall": [
+      "bower install --config.interactive=false",
+      "./node_modules/protractor/bin/webdriver-manager update"
```

Comment:
What do you think about this?

https://www.npmjs.com/package/grunt-protractor-webdriver